workers.py

```diff
@@ -84,13 +86,15 @@ def ai_service_worker(
         b64_identity: str = None,
         ) -> Thread:
     """Outlier detection."""
+    @REQUEST_TIME.time()
```

Comment:
In the graph below, the pipeline was executed 5 times over a span of 6 hours,
and the `REQUEST_TIME` seemed to jump as shown below -

0 --> 85.07 --> 91.47 --> 97.51 --> 103.64 --> 109.62

<img width="1410" alt="Screen Shot 2019-05-09 at 5 02 17 PM" src="https://user-images.githubusercontent.com/1538216/57494041-74a51d80-727c-11e9-97ba-d20f4d452248.png">

The first execution takes the longest - 85 secs, but thereafter the difference is only a few seconds.
Does this seem right?

Reply:
The time depends on data size (how many systems) and feature size.  I've tested one of the account which has 1000+ systems and the time was 79 seconds.  So you can check the metric for `data_size`